docs/experiments/fourier_checking.ipynb

```diff
@@ -32,12 +32,10 @@
    ]
```

Comment:
<p>this still looks off to me. (first line is what?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>